hal/armv7m/imxrt/10xx/imxrt10xx.c

```diff
@@ -2131,9 +2139,26 @@ void _imxrt_init(void)
 	if (*(imxrt_common.wdog2 + wdog_wcr) & (1 << 2))
 		*(imxrt_common.wdog2 + wdog_wcr) &= ~(1 << 2);
 
-	*(imxrt_common.rtwdog + rtwdog_cnt) = 0xd928c520; /* Update key */
-	*(imxrt_common.rtwdog + rtwdog_total) = 0xffff;
-	*(imxrt_common.rtwdog + rtwdog_cs) = (*(imxrt_common.rtwdog + rtwdog_cs) & ~(1 << 7)) | (1 << 5);
+	/* RTWDOG unlock update */
+	*(imxrt_common.rtwdog + rtwdog_cnt) = RTWDOG_UPDATE_KEY;
+	while(!(*(imxrt_common.rtwdog + rtwdog_cs) & (1 << 11)));
```

Comment:
Missing space after `while`. Also, moving `;` of an empty `while` to the new line is a good practice.

Reply:
Fixed